air-script/tests/pub_inputs/pub_inputs.masm

```diff
@@ -1,16 +1,70 @@
 proc.compute_evaluate_transitions
-    # constraint 0 for main
+    # integrity constraint 0 for main
```

Comment:
nit: as mentioned in one of previous comments - instead of "transition" we should use "integrity" - so, something like `compute_evaluate_integrity`. I'd also probably add "constraints" at the end - i.e., `compute_evaluate_integrity_constraints`.